docs/checkers/index.md

```diff
@@ -1169,6 +1176,27 @@ lines: 1-3
 ---
 ```
 
+PythonTA allows you to specify all the modules that you wish to allow for a particular file using the
+allowed-import-modules configuration option in the check_all function
```

Comment:
Write the option name (here and below) in backticks (`) so that they are rendered in code font.